pkg/cbfs/types.go

```diff
@@ -247,6 +247,10 @@ type BootBlockRecord struct {
 	File
 }
 
+type BootSplashRecord struct {
+	File
```

Comment:
since you have a GetFile, you may want to make this
file
not
File

This may make the race detector less likely to whine at you?

Reply:
I don't understand... isn't this just the same thing as all the others?

Reply:
and I'd want the File to be simply serializable, at least to JSON

Reply:
Yeah File looks correct especially for JSON

Reply:
Anything else from you @rminnich? Maybe something on the race detector? I'm not seeing where it may find an issue.